client/js/store/core/initStore.js

```diff
@@ -71,25 +71,32 @@ function initStore (storeModules, apiStoreModules, presetStoreModules) {
               'X-Demosplan-Procedure-Id': dplan.procedureId
             },
             successCallbacks: [
-              (response) => {
-                if (typeof response.data !== 'undefined' &&
-                typeof response.data.meta !== 'undefined' &&
-                typeof response.data.meta.messages !== 'undefined') {
-                  handleResponseMessages(response.data.meta)
+              async (success) => {
+                const response = await success.json()
+
+                const meta = response.data?.meta
+                  ? response.data.meta
+                  : response.meta || null
+                if (meta?.messages) {
+                  handleResponseMessages(meta)
                 }
+
                 return Promise.resolve(response)
               }
             ],
             errorCallbacks: [
-              (err) => {
-                const response = err.response
-                if (typeof response !== 'undefined' &&
-                typeof response.data !== 'undefined' &&
-                typeof response.data.meta !== 'undefined' &&
-                typeof response.data.meta.messages !== 'undefined') {
-                  handleResponseMessages(response.data.meta)
+              async (err) => {
```

Comment:
Nitpick: i find `error` more consistent with the `success` naming in successCallback.